src/releng_docs/projects/shipit-pulse-listener.rst

```diff
@@ -0,0 +1,27 @@
+.. shipit-pulse-listener-project:
+
+Project: shipit-pulse-listener
+==============================
+
+:contact: `Release Management`_
+
+This is a monitoring component that reacts to pulse messages, like:
+
+- `mozilla-central build`_ finished
+- new MozReview_ review request submitted
+
+It also polls for new things to do:
+
+- new Phabricator_ differential submitted
+
+When it has captured a new event, it triggers tasks:
+
+- ``shipit_code_coverage``: triggered after builds finished, to gather and
+  upload coverage data;
+- ``shipit_static_analysis``: triggered on review requests, to provide linting
+  and static analysis reports through a comment made by a bot.
```

Comment:
Could you list the Hooks classes here (defined in `listener.py`), and explain shortly what they do.

Reply:
Updated.